packages/toolkit/src/components/NewsLetterCard.tsx

```diff
@@ -0,0 +1,172 @@
+      const month = parseInt(dateParts[1], 10);
+      const day = parseInt(dateParts[2], 10);
+
+      const monthNames = [
+        "January", "February", "March", "April", "May", "June",
+        "July", "August", "September", "October", "November", "December"
+      ];
+
+      const formattedDate = `${monthNames[month - 1]} ${day}, ${year}`;
+      return formattedDate;
+    } catch (error) {
+      console.error("Error parsing date:", error);
+      return "";
+    }
+  };
+
+  const handlePrevious = () => {
+    setCurrentIndex((prevIndex) => (prevIndex === 0 ? 0 : prevIndex - 1));
+  };
+
+  const handleNext = () => {
+    setCurrentIndex((prevIndex) =>
+      prevIndex === blogPosts.length - 1 ? prevIndex : prevIndex + 1
+    );
+  };
+
+  if (blogPosts.length === 0) {
+    return <NewsLetterCardSkeleton />;
+  }
+
+  const { imageUrl, title, publishedOn, themeImgAlt, slug } = blogPosts[currentIndex];
+
+  return (
+    <div className="flex h-[450px] flex-col gap-y-2 rounded-sm border border-semantic-bg-line p-4">
+      <h2 className="mb-4 text-2xl font-bold">What's New?</h2>
+      <div className="relative h-[250px] w-full">
+        <a href={`https://www.instill.tech/blog/${slug}`} target="_blank" rel="noopener noreferrer">
+          <ImageWithFallback
+            src={imageUrl}
+            width={600}
+            height={400}
+            alt={themeImgAlt}
+            fallbackImg={
+              <Icons.Box className="h-8 w-8 stroke-semantic-fg-primary" />
+            }
+            className="h-full w-full object-cover"
+          />
+        </a>
+      </div>
+      <button
+        type="button"
+        className="my-1 w-min whitespace-nowrap rounded-sm bg-blue-100 px-2 py-2 capitalize text-semantic-accent-default product-button-button-2 hover:!underline "
+      >
+        {publishedOn}
+      </button>
+      <a href={`https://www.instill.tech/blog/${slug}`} target="_blank" rel="noopener noreferrer">
+        <p className="line-clamp-3 overflow-hidden">{title}</p>
+      </a>
+      <div className="mt-auto flex items-center justify-end">
+        <button
+          type="button"
+          onClick={handlePrevious}
+          className={`mr-2 ${currentIndex === 0 ? "cursor-not-allowed text-gray-400" : ""}`}
+          disabled={currentIndex === 0}
+        >
+          <Icons.ArrowLeft
+            className={`h-6 w-6 ${currentIndex === 0 ? "stroke-gray-400" : "stroke-[#1D2433CC]"}`}
+          />
+        </button>
+        <button
+          type="button"
+          onClick={handleNext}
+          className={`${currentIndex === blogPosts.length - 1 ? "cursor-not-allowed text-gray-400" : ""}`}
+          disabled={currentIndex === blogPosts.length - 1}
+        >
+          <Icons.ArrowRight
+            className={`h-6 w-6 ${currentIndex === blogPosts.length - 1 ? "stroke-gray-400" : "stroke-[#1D2433CC]"}`}
+          />
+        </button>
+      </div>
+    </div>
+  );
+};
+
+export default NewsLetterCard;
```

Comment:
Hi @thewbuk Let talk about the design token. When you browse our react component you can find out that we didn't use any original color scheme from Tailwind. And use our owned design-token like text-semantic-fg-primary. 

You can actually find them in the handover on Figma. Some time it will be the semantic value like this, but sometime when Dani runs out of time, he may give the color as netural-100, you can reverse check it from our design-token file 

https://github.com/instill-ai/console/blob/main/packages/design-tokens/tokens/semantic/colour.json